From 3be5f33d76d4e34b1bc2671f7dbbc6c5fc3ccadd Mon Sep 17 00:00:00 2001
From: Conrad Kostecki <conrad@kostecki.com>
Date: Sat, 29 Aug 2020 00:06:53 +0200
Subject: [PATCH] Makefile: use xunit2 as default for pytest

PytestDeprecationWarning: The 'junit_family' default
value will change to 'xunit2' in pytest 6.0.

Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d5d4866..e88277e 100644
--- a/Makefile
+++ b/Makefile
@@ -30,4 +30,4 @@ test/fixtures/modules/palaver.so: palaver.so
 .PHONY: test-integration
 test-integration: test/fixtures/modules/palaver.so
 	@mkdir -p test-reports
-	pytest --junitxml=test-reports/junit.xml
+	pytest -o junit_family=xunit2 --junitxml=test-reports/junit.xml
